check-your-phone.md

```diff
@@ -0,0 +1 @@
+You look up and suddenly, there is a woman in front of you. She tells you to get off your phone, because "You're on vacation in Paris!"
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.